web/src/composables/useLogs.ts

```
@@ -4342,6 +4412,721 @@
+        searchObj.data.queryResults.aggs = response.content.aggs;
+        const lastRecordTimeStamp = parseInt(
+          searchObj.data.queryResults.hits[0][
+            store.state.zoConfig.timestamp_column
+          ],
+        );
+        searchObj.data.queryResults.hits = response.content.hits;
+      }
+
+      if (!searchObj.meta.refreshInterval) {
+        // In page count we set track_total_hits
+        if (!queryReq.query.hasOwnProperty("track_total_hits")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)